pkg/util/kubernetes/kubelet/podwatcher.go

```diff
@@ -123,16 +128,20 @@ func (w *PodWatcher) computeChanges(podList []*Pod) ([]*Pod, error) {
 
 		// Detect changes in labels and annotations values
 		newLabelsOrAnnotations := false
+		newPhase := false
```

Comment:
nit: could you update the comment in L129 accordingly as we detect phase changes now?

Reply:
Done, makes sense!